instant-xml-macros/src/de.rs

```diff
@@ -0,0 +1,289 @@
+use crate::{get_namespaces, retrieve_attr};
+use proc_macro::TokenStream;
+use proc_macro2::{Ident, Span};
+use quote::quote;
+
+pub struct Deserializer<'a> {
+    /// Original input.
+    pub input: &'a syn::DeriveInput,
+    pub fn_deserialize: proc_macro2::TokenStream,
```

Comment:
Let's do the same treatment for imports as on the serializer side, and make all these fields private.

Reply:
done